packages/networks/src/networks.ts

```diff
@@ -63,6 +63,63 @@ export const networks = {
     protocolMultisig: undefined,
   },
   // TEST NETWORKS
+  mantleTestnetSepolia: {
+    name: 'Mantle Testnet Sepolia',
+    type: NetworkType.testnet,
+    key: 'mantleTestnetSepolia',
+    shortKey: 'mantleTestnet',
+    color: '#B026FF',
```

Comment:
I wonder if there is a better color to represent mantle?  Maybe black?

Reply:
I made it a darkish green to match a color that's prominent in their branding